src/pubsub-engine.ts

```diff
@@ -1,5 +1,5 @@
 export interface PubSubEngine {
-  publish(triggerName: string, payload: any): boolean;
+  publish(triggerName: string, payload: any): boolean | Promise<void>;
```

Comment:
I think we should drop the `boolean` since adding the `Promise` already introduces a breaking change.

Reply:
👍